src/AvtaleSide/steg/BeregningTilskudd/visningTilskuddsperioder/TilskuddsperiodeTabellVtao.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { TilskuddsPeriode } from '@/types/avtale';
+import { addDays, getYear, isWithinInterval } from 'date-fns';
+import { BodyShort, Table } from '@navikt/ds-react';
+import { formaterDato, formaterPeriode } from '@/utils/datoUtils';
+import EtikettStatus from '@/BeslutterSide/EtikettStatus';
+import { formaterPenger, IKKE_NOE_BELOP_TEGN } from '@/utils';
+
+const TilskuddsperiodeRadVtao: React.FC<{
+    avtaleOpprettet: Date;
+    erNavAnsatt: boolean;
+    periode: TilskuddsPeriode;
+    kreverOppfølgingDato?: Date | undefined;
+}> = ({ avtaleOpprettet, erNavAnsatt, periode, kreverOppfølgingDato }) => {
```

Comment:
Det er lurt å bruke `const` for å destructure props, hvis ikke blir de default variabler:

```suggestion
interface Props {
    avtaleOpprettet: Date;
    erNavAnsatt: boolean;
    periode: TilskuddsPeriode;
    kreverOppfølgingDato?: Date | undefined;
}

const TilskuddsperiodeRadVtao = (props: Props) => {
    const { avtaleOpprettet, erNavAnsatt, periode, kreverOppfølgingDato } = props;
```